File: PRAK603-2210817220001-AjengDiahPramesti.c
#include <stdio.h>

int main()
{
    int baris, baris2, a, nilai, hasil;
    scanf("%d %d", &baris, &baris2);
    int angka[baris], angka2[baris2];
    if (baris == baris2) {
    for (a = 0; a < baris; a++ ){ 
        scanf("%d", &nilai);
        angka[a] = nilai;
    }
    for (a = 0; a < baris2; a++ ){ 
        scanf("%d", &nilai);
        angka2[a] = nilai;
        }
    for (a = 0; a < baris; a++ ){ 
        hasil = angka[a] * angka2[a];
        printf("%d ", hasil) ;
        }
    }
    else {
        printf("Jumlah tidak sama");
    }
}

File: PRAK603-2210817220001-AjengDiahPramesti.py
baris, baris2 = map(int, input().split())
if baris != baris2 :
    print("Jumlah tidak sama")
else :
    nilai = list(map(int, input().split()))
    nilai2 = list(map(int, input().split()))
    matriks = []
    matriks2 = []
    a = 0
    matriks.append(nilai[a : a + baris])
    matriks2.append(nilai2[a : a + baris2])
    for i in range(0, baris) :
        print(matriks[0][i] * matriks2[0][i], end=' ')